src/pages/publish-online-guidence/code snippets/GuidencePageExampleCodeSnippet.tsx

```diff
@@ -0,0 +1,174 @@
+export const GuidencePageExampleCodeSnippet = () => {
+  return (
+    <pre>
+      <code>
+        <span className="hljs-tag">
+          <span className="hljs-attr">&lt;html&gt;</span>
+        </span>
+        <br />
+        &nbsp;&nbsp;
+        <span className="hljs-tag">
+          <span className="hljs-attr">&lt;head&gt;</span>
+        </span>
+        <br />
+        &nbsp;&nbsp;&nbsp;
+        <span className="hljs-tag">
+          <span className="hljs-attr">&lt;title&gt;</span>
+          My first HTML page
+          <span className="hljs-attr">&lt;/title&gt;</span>
+        </span>
+        <br />
+        &nbsp;&nbsp;&nbsp;
+        <span className="hljs-tag">
+          <span className="hljs-attr">&lt;link</span>
+        </span>
+        <br />
+        &nbsp;&nbsp;&nbsp;&nbsp;
+        <span className="hljs-tag">
```

Comment:
I was thinking we'd make convert the CSS file to modules, i.e.:
- Rename the CSS file to `PublishOnlineGuidencePage.module.css`
- Update the import: `import styles from "./PublishOnlineGuidencePage.module.css";`
- Then we can use the class like `className={styles.hljsTag}

A couple of other notes:
- Switching to CSS modules, I'd probably move the CSS imports into the snippet files (and move the CSS file itself)
- I would replace space in this folder name with a hyphen (i.e. `code-snippets`)

I actually have this locally because I just wanted to double check so I could push it for you if that's better?

Reply:
Discussed offline, pushed these changes